src/components/Card/Card.tsx

```diff
@@ -90,7 +90,7 @@ function Card({
 
   return (
     <Link
-      to={url ?? ''}
+      to={disabled ? '/' : url}
```

Comment:
I think this will only work on the home page where we want to route to the root (`/`).  You should be able to set the `to` prop to the url unconditionally, but then set disabled on the Link itself.
```suggestion
      to={url}
      disabled={disabled}
```